Core/HLE/sceAtrac.cpp

```diff
@@ -374,9 +374,11 @@ struct Atrac {
 #else
 		// Future versions may add other things to free, but avcodec_free_context didn't exist yet here.
 		avcodec_close(pCodecCtx);
-		av_freep(&pCodecCtx->extradata);
-		av_freep(&pCodecCtx->subtitle_header);
-		av_freep(&pCodecCtx);
+        if ( pCodecCtx != NULL ) {
+            av_freep(&pCodecCtx->extradata);
+            av_freep(&pCodecCtx->subtitle_header);
+            av_freep(&pCodecCtx);
+        }
```

Comment:
Indent with hard tabs, please.  If the diff on GitHub looks wrong, the indentation is incorrect.

-[Unknown]